src/iterable/takeWhile.js

```diff
@@ -11,7 +11,7 @@
  *
  * ```javascript
  * [1,2,3,4,1,2,3,4]::takeWhile(function () {
- *   return this &lt; 4;
```

Comment:
Heh, this is what happens when you accidentally `rm -rf .` and reverse generate function declarations with the JSDoc from the generated HTML documentation x)